apps/www/src/state/plugin/operationAlerts.js

```diff
@@ -1,15 +1,35 @@
+import { rdfs } from '@tpluscode/rdf-ns-builders'
+import { html } from 'lit'
+
 export const operationAlerts = {
   model: {
     state: {},
-    reducers: {},
+    reducers: {
+      lastError(state, lastError) {
+        return { ...state, lastError }
+      },
+    },
     effects(store) {
-      const { notifications } = store.getDispatch()
+      const dispatch = store.getDispatch()
 
       return {
-        'operation/failed': ({ response }) => {
+        'operation/failed': ({ response, error }) => {
+          if (error) {
+            const { pointer } = error
+            dispatch.notifications.show({
+              content: html`
+                <strong>${pointer.out(rdfs.label).value}</strong><br>
+                <sl-button outline @click="${() => dispatch.notifications.errorDialog(pointer)}">See details</sl-button>
+              `,
+              variant: 'danger',
+              autoHide: false,
+            })
```

Comment:
See how a notification can have rich content, with action trigger for opening the SHACL details